PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -849,4 +855,14 @@ export class ParaReportPublishService {
 
         return uuid.v5(violType + ruleId + msg + severity + lineHash + uri + order, namespace);
     }
+
+    private checkDuplicatedStaticAnalysisReportName = (sourcePath: string): boolean => {
+        let filename = path.basename(sourcePath);
+        if (this.originalStaticAnalysisReportMap.has(filename)) {
+            tl.warning(`Static analysis report ${sourcePath} is skipped, it has the same name with report ${this.originalStaticAnalysisReportMap.get(filename)} which has already processed.Try to use a different report file name.`);
```

Comment:
Use message: Can't handle report with the duplicate name '${filename}', which may cause processing issues. ${sourcePath} is skipped.